tests/hla/test_align.py

```diff
@@ -0,0 +1,215 @@
+          * A single astrometric catalog was obtained with both GAIA and non-GAIA
+            (PanSTARRS?) sources for the entire combined field-of-view using the GSSS
+            server.
+              * These tests assume combined input exposures for each test have enough
+                astrometric sources from the external catalog to perform a high quality
+                fit.
+          * This test only determines the fit between sources
+            extracted from the images by Tweakreg and the source positions included in
+            the astrometric catalog.
+          * The WCS information for the input exposures do not get updated in this test.
+          * No mosaic gets generated.
+
+        Success Criteria:
+          * Success criteria hard-coded for this test represents 10mas RMS for the
+            WFC3 images based on the fit of source positions to the astrometric catalog
+            source positions.
+              * RMS values are extracted from optional shiftfile output from `tweakreg`
+              * Number of stars used for the fit and other information is not available
+                with the current version of `tweakreg`.
+
+        The environment variable needs to be set in the following manner:
+            export TEST_BIGDATA=https://bytesalad.stsci.edu/artifactory/
+            OR
+            export TEST_BIGDATA=/Users/YourNameHere/TestDataDirectory/
+     
+        This test file can be executed in the following manner:
+            $ pytest -s --bigdata test_align.py >& test_align_output.txt &
+            $ tail -f test_align_output.txt
+
+    """
+
+    ref_loc = ['truth']
+
+    def test_align_ngc188(self):
+        """ Verify whether NGC188 exposures can be aligned to an astrometric standard.
+
+        Characeteristics of this test:
+          * Input exposures include both ACS and WFC3 images of the same general field-of-view
+            of NGC188 suitable for creating a combined mosaic using both instruments.
+        """
+        self.input_repo = 'hst-hla-pipeline'
+        self.input_loc = 'mosaic_ngc188'
+        totalRMS = 0.0
+        input_filenames = ['iaal01hxq_flc.fits', 'iaala3btq_flc.fits',
+                            'iaal01hyq_flc.fits', 'iaala3bsq_flc.fits',
+                            'j8boa1m8q_flc.fits', 'j8boa1m4q_flc.fits',
+                            'j8boa1maq_flc.fits', 'j8boa1m6q_flc.fits']
+
+        data_path = ['hst-hla-pipeline','dev','mosaic_ngc188']
+        try:
+            datasetTable = alignimages.perform_align(input_filenames,archive=False,clobber=True,debug=False,
+                update_hdr_wcs=False,print_fit_parameters=True,print_git_info=False,output=False,regtest=True,
+                test_data_path=data_path)
+
+            # Examine the output table to extract the RMS for the entire fit and the compromised information
+            if (datasetTable):
+                totalRMS = datasetTable['total_rms'][0]
+
+        except Exception:
```

Comment:
These `traceback` blocks prevent `pytest` from generating meaningful tracebacks after a failed test. I'm only commenting on this one but it applies to all of them.

Use:
```python
# Fail if exception of `SomeExceptionHere` is not raised
with not pytest.raises(SomeExceptionHere):
    # code here

# Pass if exception of type `SomeExceptionHere` is raised:
with pytest.raises(SomeExceptionHere):
    # code here
```

In all other cases let pytest parse the traceback and record it correctly.